autonomy/data/Dockerfiles/tendermint/app.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
```

Comment:
Right now we'll have to keep this utility scripts as a part of the data folder, We'll port them properly later